src/components/subscribe/constants.js

```diff
@@ -1,7 +1,8 @@
 export const Wording = {
   TITLE: "Rămâi informat",
   SUB_TITLE:
-    "Abonează-te la newsletter și îți trimitem zilnic ultimele informații!",
+    "Abonează-te la newsletter și te ținem la curent cu cele mai importante \
+    informații și cu toate soluțiile digitale pe care le dezvoltăm în cadrul acestui proiect.",
```

Comment:
nit
```suggestion
    `Abonează-te la newsletter și te ținem la curent cu cele mai importante
    informații și cu toate soluțiile digitale pe care le dezvoltăm în cadrul acestui proiect.`,
```

Reply:
@moonflare initially did that ran `npm run lint:fix` and it changed it back to double quotes since it doesn't contain interpolation.